cosmos/providers/dbt/render.py

```diff
@@ -25,7 +25,7 @@ def render_project(
     test_behavior: Literal["none", "after_each", "after_all"] = "after_each",
     emit_datasets: bool = True,
     conn_id: str = "default_conn_id",
-    dbt_tags: List[str] = [],
+    select: Dict = None,
```

Comment:
could/should this be something like:
```suggestion
    select: Dict[str, List[str]] = None,
```

Reply:
Added.